lambda/src/firebase-authorizer/firebase-authorizer.js

```diff
@@ -116,29 +117,26 @@ const checkUserAuthorization = async event => {
   }
 }
 
+const _handler = async event => {
+  initNFA({
+    firebaseProjectId: process.env.LAMBDA_FIREBASE_PROJECT_ID,
+    firebasePrivateKey: decryptedFirebasePrivateKey.replace(/\\n/g, '\n'),
+    firebaseClientEmail: process.env.LAMBDA_FIREBASE_CLIENT_EMAIL,
+    firebaseDatabaseURL: process.env.LAMBDA_FIREBASE_DATABASE_URL,
+    firebasePublicAPIKey: process.env.FIREBASE_PUBLIC_API_KEY,
+    cookieKeys: [await decryptValue(process.env.COOKIE_SECRET_20220711)],
```

Comment:
This will call `decryptValue` on each invocation, which probably isn't what you want?